components/director/internal/domain/labeldef/service.go

```diff
@@ -234,11 +240,22 @@ func (s *service) GetAvailableScenarios(ctx context.Context, tenantID string) ([
 }
 
 // AddDefaultScenarioIfEnabled adds DEFAULT scenario if defaultScenarioEnabled
-func (s *service) AddDefaultScenarioIfEnabled(ctx context.Context, labels *map[string]interface{}) {
+func (s *service) AddDefaultScenarioIfEnabled(ctx context.Context, tnt string, labels *map[string]interface{}) {
 	if labels == nil || !s.defaultScenarioEnabled {
 		return
 	}
 
+	// do not add scenario to subaccounts
+	btm, err := s.tenantRepo.Get(ctx, tnt)
```

Comment:
can we do this a bit more generically - for example instead of check for tenant type, check if the label definition for this scenario has a schema that contains the Default scenario

Reply:
As discussed, we will remain the following check for the hotfix. In the next sprint we will evaluate whether we could drop this piece of code.